html/changelogs/RunaDacin-GenderIdentityToggle.yml

```diff
@@ -0,0 +1,5 @@
+author: Runa-Dacino
```

Comment:
We don't use the changelogs. Those are for Polaris. You can delete this entirely.